server/lib/availabilitySync.ts

```diff
@@ -73,29 +73,50 @@ class AvailabilitySync {
         });
       }
 
-      if (mediaServerType === MediaServerType.PLEX) {
-        if (admin && admin.plexToken) {
-          this.plexClient = new PlexAPI({ plexToken: admin.plexToken });
-        } else {
-          logger.error('Plex admin is not configured.');
-        }
-      } else if (
-        mediaServerType === MediaServerType.JELLYFIN ||
-        mediaServerType === MediaServerType.EMBY
-      ) {
-        if (admin) {
-          this.jellyfinClient = new JellyfinAPI(
-            getHostname(),
-            admin.jellyfinAuthToken,
-            admin.jellyfinDeviceId
-          );
+      switch (mediaServerType) {
```

Comment:
Why change the if statement with a switch?

Reply:
Because when i add in the try block for connection checking, it ended up getting really nested and hard to read. Should i change it back?

Reply:
No no it's ok, I was just wondering why did you change it